packages/react-scripts/scripts/init.js

```diff
@@ -208,6 +208,12 @@ module.exports = function (
     extends: 'react-app',
   };
 
+  // Setup the prettier config
+  appPackage.prettier = {
+    arrowParens: 'avoid',
+    singleQuote: true,
```

Comment:
This is what I would recommend to beginners of React and Prettier now, though I'd like opinions before committing it:
```suggestion
    semi: false,
    trailingComma: 'all',
```

Reply:
Alternatively, I can start a config package, and we can discuss options there